Database/Corrections/sodQuestFixes.lua

```diff
@@ -2928,6 +2928,18 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -402852,
             [questKeys.zoneOrSort] = sortKeys.PRIEST,
         },
+        [90197] = {
+            [questKeys.name] = "Between the Eyes",
+            [questKeys.startedBy] = {{2011,2012,2013},{407352}},
```

Comment:
Same as before, we should remove the NPCs from this rune, to reduce icons.

```suggestion
            [questKeys.startedBy] = {nil,{407352}},
```